src/components/Thread/Thread.js

```diff
@@ -23,9 +24,25 @@ class Thread extends Component {
         this.props.dispatch(threadActions.loadThread(id));
     }
 
+    componentDidMount = () => {
+        
+    }
+
+    componentWillUnmount() {
+        this.props.dispatch(threadActions.exitThreadView());
+        this.props.dispatch(threadActions.exitPostsView());
+    }
+
     render() {
         return (
-            <div className='container'>
+            <div
+                style= {{
```

Comment:
eh we should still use css for this stuff, scott or i can go through and clean though

Reply:
Yeah we need to either figure out if we are going to use scss, css modules, or something akin. Go for it :)